exotic.py

```diff
@@ -1637,26 +1551,31 @@ def parse_args():
 
         # Handle AAVSO Formatting
         if fileorcommandline == 1:
-            AAVSOoutput = user_input('Do you have an AAVSO Observer Account? (y/n): ', type_=str, val1='y', val2='n')
-            if AAVSOoutput == 'n':
+            infoDict['aavsoopt'] = user_input('Do you have an AAVSO Observer Account? (y/n): ', type_=str, val1='y', val2='n')
+            if infoDict['aavsoopt'] == 'n':
                 AAVSOBool = False
             else:
                 AAVSOBool = True
                 # userNameEmails = str(input('Please enter your name(s) and email address(es) in the format: Your Name (youremail@example.com), Next Name (nextemail@example.com), etc.  '))
-                userCode = str(input('Please enter your AAVSO Observer Account Number: '))
-                secuserCode = str(input('Please enter your comma-separated secondary observer codes (or type none if only 1 observer code): '))
-            cameraType = str(input("Please enter your camera type (CCD or DSLR): "))
-            binning = str(input('Please enter your pixel binning: '))
-            exposureTime = str(input('Please enter your exposure time (seconds): '))
-            filterName = str(input('Please enter your filter name from the options at '
+                infoDict['aavsonum'] = str(input('Please enter your AAVSO Observer Account Number: '))
```

Comment:
Recommend adding `(please input n/a if you do not currently have an account)` as we should always build the AAVSO file from now on

Reply:
Done!